### PR TITLE
BLOG - Apache Cassandra 5.0 Features: Trie Memtables and Trie-Indexed SSTables

### PR DESCRIPTION
**Pull request by nonstopdtop**
patch by Diogenese Topper, Andrés de la Peña; reviewed by -- for CASSANDRA-18900

### CODE REVIEW

File: site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-5.0-Features-Trie-Memtables-and-Trie-Indexed-SSTables.adoc [hunk 1]
```diff
@@ -0,0 +1,73 @@
+= Apache Cassandra 5.0 Features: Trie Memtables and Trie-Indexed SSTables
+:page-layout: single-post
+:page-role: blog-post
+:page-post-date: November 8, 2023
+:page-post-author: Andrés de la Peña
+:description: New dynamic data masking capabilities are a feature available in the coming Apache Cassandra 5.0.
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-5.0-Features-Trie-Memtables-and-Trie-Indexed-SSTables.adoc]: needs update

File: site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-5.0-Features-Trie-Memtables-and-Trie-Indexed-SSTables.adoc [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+:keywords: 
+
+__Apache Cassandra 5.0 is the project’s major release for 2023, and it promises some of the biggest changes for Cassandra to-date. After more than a decade of engineering work dedicated to stabilizing and building Cassandra as a distributed database, we now look forward to introducing a host of exciting features and enhancements that empower users to take their data-driven applications to the next level - including machine learning and artificial intelligence.__
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-5.0-Features-Trie-Memtables-and-Trie-Indexed-SSTables.adoc]:
```suggestion
__Apache Cassandra 5.0 is the project’s major release for 2023, and it promises some of the biggest changes for Cassandra to-date. After more than a decade of world class engineering building Cassandra as the safest most stable distributed database, we are witness now to a new chapter of innovation introducing a host of exciting features and enhancements that empower users to take their data-driven applications to the next level - including machine learning and artificial intelligence.__
```

**Review by michaelsembwever — CHANGES_REQUESTED** [review]
small changes requested.

otherwise review complete and everything is good to go.

**Comment by nonstopdtop** [comment]
Requested changes made per your review @michaelsembwever

**Review by michaelsembwever — APPROVED** [review]

**Comment by michaelsembwever** [comment]
committed https://github.com/apache/cassandra-website/commit/30cc955133045bdc5029867b605238df3a331dfd

**Pull request closed without merging**